flybywiresim/a32nx
# ci: recognise external downloads in installer

## Summary of Changes

- add install.json to the distributed zip so the installer recognizes it as an installation

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): Foxtrot Sierra#6420

## Testing instructions

- download PR artifact and install
- see the installer recognize it as updatable


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
